Makefile

```diff
@@ -126,13 +123,6 @@ docker-vue:
 docker-test:
 	docker-compose -f docker-compose.test.yml up --remove-orphans
 
-push-base: clean compile docker-base
-	docker push statping-ng/statping-ng:base
-
-push-vue: clean compile docker-base docker-vue
-	docker push statping-ng/statping-ng:base
-	docker push statping-ng/statping-ng:vue
-
```

Comment:
you removed `push-vue` (which I don't think is used by anything) but kept `docker-vue`. I think we should remove either both or none of them

Reply:
I would remove both, but that's mainly because I don't understand what to do with them.

Reply:
removed in new commit